pyproject.toml

```
@@ -15,10 +15,13 @@ description = "Design and analysis tools for LIBRA project"
 license = {file = "LICENSE"}
 requires-python = ">=3.6"
 dynamic = ["version"]
-dependencies = ["numpy", "pint", "scipy", "matplotlib", "sympy", "pandas"]
+dependencies = ["numpy", "pint", "scipy", "matplotlib", "sympy", "pandas", "openmc>=0.14.0", "h5py"]
 
 [project.optional-dependencies]
 tests = ["pytest>=5.4.3", "pytest-cov", "nbconvert", "ipykernel"]
 
 [tool.setuptools_scm]
-write_to = "libra_toolbox/_version.py"
\ No newline at end of file
+write_to = "libra_toolbox/_version.py"
+
+[tool.setuptools.package-data]
+mvng_source = ["libra_toolbox/neutronics/neutron_source_diamond.h5"]
```

Since you modified the name of the h5 file, you need to modify this too!
Would something like this be safer and more generic?

```suggestion
mvng_source = ["*.h5"]
```